tools/testdata-scripts/src/bin/make-testdata.rs

```diff
@@ -34,9 +14,9 @@ fn main() {
         .unwrap();
 
     let source_data = SourceData::default()
-        .with_cldr(paths::cldr_json_root(), CldrLocaleSubset::Full)
+        .with_cldr_for_tag(&icu_testdata::versions::cldr_tag(), CldrLocaleSubset::Full)
         .unwrap()
-        .with_icuexport(paths::icuexport_toml_root())
+        .with_icuexport_for_tag(&icu_testdata::versions::icu_tag())
```

Comment:
The title of this PR should be "Make testdata datagen download its sources directly"; that's the bigger change being made. The fact that it gets a few more keys is a side-effect.